x-pack/plugins/fleet/server/services/epm/elasticsearch/template/template.ts

```diff
@@ -179,6 +195,55 @@ function _generateMappings(
       // If type is not defined, assume keyword
       const type = field.type || 'keyword';
 
+      if (field.runtime !== undefined) {
+        const path = ctx.groupFieldName ? `${ctx.groupFieldName}.${field.name}` : field.name;
+        let runtimeFieldProps: Properties = getDefaultProperties(field);
+
+        // Is it a dynamic template?
+        if (type === 'object' && field.object_type) {
+          const pathMatch = path.includes('*') ? path : `${path}.*`;
+
+          let dynProperties: Properties = getDefaultProperties(field);
+          let matchingType: string | undefined;
+          switch (field.object_type) {
+            case 'keyword':
+              dynProperties.type = field.object_type;
+              matchingType = field.object_type_mapping_type ?? 'string';
+              break;
+            case 'double':
+            case 'long':
+            case 'boolean':
+              dynProperties = {
+                type: field.object_type,
+                time_series_metric: field.metric_type,
+              };
+              matchingType = field.object_type_mapping_type ?? field.object_type;
+            default:
+              break;
+          }
```

Comment:
Similar to what it is done in this block: https://github.com/elastic/kibana/blob/9522282637e46b81b0a0805ea88e0f4a5254d63d/x-pack/plugins/fleet/server/services/epm/elasticsearch/template/template.ts#L188-L213

but as runtime fields support fewer types, there were removed the ones not supported

https://www.elastic.co/guide/en/elasticsearch/reference/8.8/runtime-mapping-fields.html#runtime-mapping-fields

Reply:
Should we support the other supported ones like `ip`, `geo_point` ?